library/Notifications/Widget/ItemList/EscalationConditionList.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/* Icinga Notifications Web | (c) 2023 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Notifications\Widget\ItemList;
+
+use ipl\Html\BaseHtmlElement;
+
+class EscalationConditionList extends BaseHtmlElement
+{
+    protected $defaultAttributes = ['class' => 'options'];
```

Comment:
I'm not a fan of such generic class names. The problem with these is sometimes that they get undesired styles applied due to some random rule defined somewhere else. So please use a less generic name. Such as `escalation-condition-list` :wink: